packages/css-library/dist/tokens/css/variables.css

```diff
@@ -4,7 +4,11 @@
  */
 
 :root {
+  --xsmall-screen: 320px;
   --small-screen: 481px;
+  --medium-screen: 640px;
+  --small-desktop-screen: 1008px;
+  --medium-large-screen: 640px;
```

Comment:
See the [comment below](https://github.com/department-of-veterans-affairs/component-library/pull/1042/files#r1471743668) about this duplicate 640px value.